docs/setup-cluster/on-prem/requirements.rst

```diff
@@ -149,4 +150,4 @@ are only able to run CPU-based workloads.
 Install on Windows Subsystem for Linux (WSL)
 ============================================
 
-Follow the steps for :ref:`installing Docker on Linux <install-docker-on-linux>`.
+Follow the steps for :ref:installing Docker on Linux <install-docker-on-linux>.
```

Comment:
Seems like this ref is broken?

Reply:
already fixed with https://github.com/determined-ai/determined/pull/8836

Reply:
Right, but this is reverting that change. Just add the ` back in.

Reply:
good point